pkg/controllers/mcs/mcs_controller.go

```diff
@@ -0,0 +1,397 @@
+/*
+Copyright 2021 The Karmada Authors.
```

Comment:
```suggestion
Copyright 2023 The Karmada Authors.
```
Coryright year should be the year first created this file.